ROLES.md

```diff
@@ -159,11 +187,9 @@ approver in an OWNERS file:
     added to `reviewers` section is at the discretion of an approver after
     enough evidence of quality contributions.
 
-- Reviewed or merged at least 30 PRs to the codebase.
-
-- Nominated by an area lead (with no objections from other leads).
+- Reviewed at least 30 PRs to the codebase.
 
-  - Done through PR to update an OWNERS file.
+- Nominated by an a WG lead (with no objections from other leads).
```

Comment:
Do we need the 3 month tenure requirement (line 179) here, or is "review at least 10 substantial PRs" sufficient?

I'd prefer to remove the time gates if possible, and focus on contribution.

Reply:
I think that's something we can discuss in a joint toc/steering session; do you think it needs to block this PR? we can always amend the rules later if we have consensus.

Reply:
We discussed this in a steering work session and decided to keep the 3 month requirement for now to ensure folks have a solid amount of burn-in time to develop their familiarity with the project and develop their judgement.

Reply:
We can always amend the criteria in the future if we feel like the bar is too high.